examples/seesaw_rotary_neopixel.py

```diff
@@ -0,0 +1,47 @@
+"""I2C rotary encoder NeoPixel color picker and brightness setting example."""
+import board
+from adafruit_seesaw import seesaw, neopixel, rotaryio, digitalio
+try:
+    import _pixelbuf
+except ImportError:
+    import adafruit_pypixelbuf as _pixelbuf
+
+# For use with the STEMMA connector on QT Py RP2040
+# import busio
+# i2c = busio.I2C(board.SCL1, board.SDA1)
+# seesaw = seesaw.Seesaw(i2c, 0x36)
+
+seesaw = seesaw.Seesaw(board.I2C(), 0x36)
+
+encoder = rotaryio.IncrementalEncoder(seesaw)
+switch = digitalio.DigitalIO(seesaw, 24)
+
+pixel = neopixel.NeoPixel(seesaw, 6, 1)
+pixel.brightness = 0.5
+
+last_position = -1
+color = 0  # start at red
+
+while True:
+    position = encoder.position
+
+    if position != last_position:
+        print(position)
+
+        if switch.value:
```

Comment:
If you rotate the encoder, and then wait a bit before you press the switch I don't believe this code executes. Is that the intended behavior or should this `if switch.value` call be outside of the `if position != last_position`, or am I not reading this code correctly (I do need to top off my coffee).

Reply:
The code executes when I try to replicate what you're suggesting. I'm not sure if I'm simply not doing it in the manner you're suggesting or if the code is fine. I'm going to merge this, we can continue the discussion if you have suggestions and I can make changes if needed in another PR.